source/07-processing/11-restating-data.rst

```diff
@@ -0,0 +1,20 @@
+.. The COUNTER Code of Practice Release 5 © 2017-2023 by COUNTER
+   is licensed under CC BY-SA 4.0. To view a copy of this license,
+   visit https://creativecommons.org/licenses/by-sa/4.0/
+
+Restating data
+--------------
+
+
+Retrospective Reporting of Errors
+"""""""""""""""""""""""""""""""""
+
+If errors are identified in COUNTER reports, the report provider must correct the errors within three months of their discovery. The report provider must alert the Project Director so that a notification may be included in the COUNTER Registry. Report providers should also directly inform their customers of the corrections.
```

Comment:
COUNTER Project Director, maybe with email address? This should be consistent thoughout the CoP.